package.json

```diff
@@ -23,6 +23,7 @@
     "@typescript-eslint/parser": "5.21.0",
     "async-retry": "1.2.3",
     "buffer-replace": "1.0.0",
+    "create-svelte": "2.0.0-next.198",
```

Comment:
Would it be better to remove this exact version and instead run `npm create svelte@latest` like the docs say here?

https://www.npmjs.com/package/create-svelte

That way it would also use the latest version each run of cron?

Reply:
Yes and no -- if you read the script, you'll see that it upgrades `create-svelte` if a newer version is available (and includes that upgrade as part of the PR) -- but the problem is, `create-svelte` is an interactive prompt and doesn't support command-line flags. Instead, it also exports a programmatic interface. I actually asked Rich about this and offered to add support, and some of the other maintainers dug up this: https://github.com/sveltejs/kit/pull/6117#issuecomment-1221323822

Which TBH I actually agree with.

Reply:
Does this mean someone needs to manually bump that version each time?

Since the programmatic API will be subject to semver once we come out of pre-release mode, we could pin it to a major?